### PR TITLE
feat : Added Sponsor Models

### PR DESCRIPTION
**Pull request by shreyasnbhat**
Added sponsor models.

### CODE REVIEW

**Comment by shreyasnbhat** [comment]
@iamareebjamal Please review.

File: src/main/java/openevent/Application.java [hunk 1]
```diff
@@ -1,24 +1,33 @@
...
+
 import openevent.model.Event;
 import openevent.model.Microlocation;
+<<<<<<< a595619ec0d15e6233263353fcb7a3058e081946
 import openevent.model.SessionType;
 import openevent.model.Track;
 import openevent.model.api.ApiEvent;
 import openevent.model.api.ApiMicrolocation;
 import openevent.model.api.ApiSessionType;
+=======
```
Comment (iamareebjamal) [inline src/main/java/openevent/Application.java]: some merge conflicts are still there

**Review by iamareebjamal — CHANGES_REQUESTED** [review]

File: src/main/java/openevent/model/api/ApiSponsor.java [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.github.jasminb.jsonapi.annotations.Id;
+import com.github.jasminb.jsonapi.annotations.Type;
+
+@Type("sponsor")
+public class ApiSponsor {
+	
+	@Id
+	@JsonProperty
```
Comment (arp95) [inline src/main/java/openevent/model/api/ApiSponsor.java:11]: @iamareebjamal 
what is the use of this class?
we are using sponser.java as a model right, so the object sponser will be of this type and we will take data through that.
what exactly does this class do?

File: src/main/resources/sponsors.json [hunk 3]
```diff
@@ -0,0 +1,19 @@
+[{
+        "id": 10, 
+        "name": "Python", 
+        "logo": "/images/sponsors/Python_10.png",
```
Comment (arp95) [inline src/main/resources/sponsors.json:4]: shouldnt this be "logo-url", according to the nextgen api format?

File: src/main/java/openevent/model/api/ApiSponsor.java (hunk 2 above)
Comment (iamareebjamal) [reply src/main/java/openevent/model/api/ApiSponsor.java:11]: This is new API class

File: src/main/resources/sponsors.json (hunk 3 above)
Comment (iamareebjamal) [reply src/main/resources/sponsors.json:4]: nextgen api models are present in /api/ folder

File: src/main/java/openevent/Application.java (hunk 1 above)
Comment (shreyasnbhat) [reply src/main/java/openevent/Application.java]: Will fix.

**Comment by shreyasnbhat** [comment]
@iamareebjamal Ready for review.

File: src/main/resources/api/sponsors.json [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+          }
+        }
+      },
+      "attributes": {
+        "description": "Google",
+        "level": "1",
+        "url": "http://google.com",
+        "type": "Gold",
+        "logo-url": "http://placehold.it/200X200",
+        "name": "Goolgle is a Tech Corp"
```
Comment (Shailesh351) [inline src/main/resources/api/sponsors.json:43]: I think you have added description in name and name in description :sweat_smile:

Comment (shreyasnbhat) [reply src/main/resources/api/sponsors.json:43]: I guess it doesn't matter when the main motive is to do model conversion. 
Will keep this in mind the next time. 👍

**Review by Shailesh351 — APPROVED** [review]

**Review by arp95 — APPROVED** [review]

**Pull request merged**